cmd/pipecd/server.go

```diff
@@ -188,6 +189,7 @@ func (s *server) run(ctx context.Context, input cli.Input) error {
 	is := insightstore.NewStore(fs)
 	cmdOutputStore := commandoutputstore.NewStore(fs, input.Logger)
 	statCache := rediscache.NewHashCache(rd, defaultPipedStatHashKey)
+	unregisteredAppsCache := memorycache.NewTTLCache(ctx, 24*time.Hour, 3*time.Hour)
```

Comment:
Memory cache will not work for us in this case so we have to use `redis` cache instead.
Request from Piped and Web can be routed and handled by a random server instance so memory cache is useless.

Reply:
For sure